lib/solidus_graphql_api/types/ransack_json.rb

```
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module SolidusGraphqlApi
+  module Types
+    class RansackJson < GraphQL::Types::JSON
+      def self.coerce_input(value, _ctx)
+        value.each do |k, v|
+          begin
+            case v
+            when String
+              value[k] = Integer(GraphQL::Schema::UniqueWithinType.decode(v).last)
+            when Array
+              v.each_with_index do |child_field, index|
+                v[index] = Integer(GraphQL::Schema::UniqueWithinType.decode(child_field.to_s).last)
+              end
+            end
+          rescue ArgumentError
```

do we really need to discard an exception in order to handle this flow?